fetch.bs

```diff
@@ -2049,6 +2051,29 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partition>HTTP Cache partition</h3>
+
+<p>To determine the associated <dfn export id=http-cache>HTTP cache</dfn> partition, given
+<var>request</var>, run these steps:
+
+<ol>
+ <li>
+  <p>If <var>request</var>'s <a for=request>client</a> is null, terminate these steps.</li>
+
+ <li>
+  <p>Let <var>environmentSettings</var> be <var>request</var>'s <a for=request>client</a>.</li>
+
+ <li>
+  <p>Let <var>topLevelOrigin</var> be <var>environmentSettings</var>'s top-level origin and
```

Comment:
Before we merge this you'll want to change this to `<a for="environment settings object">top-level origin</a>`. But, don't do that until https://github.com/whatwg/html/pull/4966 has merged and had a day to propagate into the linking database, as otherwise it will just cause spec compilation errors.

Reply:
Within 24 hours of this comment, this (and the blow one about obtaining an agent cluster key) should be fixable, since the HTML PR merged.